pallets/fees/src/migration.rs

```diff
@@ -1,102 +0,0 @@
-// Copyright 2022 Parity Technologies (UK) Ltd.
-// This file is part of Centrifuge (centrifuge.io) parachain.
-
-// Cumulus is free software: you can redistribute it and/or modify
-// it under the terms of the GNU General Public License as published by
-// the Free Software Foundation, either version 3 of the License, or
-// (at your option) any later version.
-
-// Cumulus is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-// GNU General Public License for more details.
-
-// You should have received a copy of the GNU General Public License
-// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
-
-use codec::{Decode, Encode};
-#[cfg(feature = "try-runtime")]
-use frame_support::ensure;
-use frame_support::{log, traits::Get, weights::Weight, Blake2_256};
-
-use super::*; // Not in prelude for try-runtime
-
-#[derive(Encode, Decode, Default, Clone, PartialEq, TypeInfo)]
-#[cfg_attr(feature = "std", derive(Debug))]
-struct Fee<Hash, Balance> {
-	key: Hash,
-	price: Balance,
-}
-
-#[frame_support::storage_alias]
-type Fees<T: Config> = StorageMap<
-	Pallet<T>,
-	Blake2_256,
-	<T as frame_system::Config>::Hash,
-	Fee<<T as frame_system::Config>::Hash, BalanceOf<T>>,
->;
-
-pub mod fee_balances {
-	use super::*;
-
-	#[cfg(feature = "try-runtime")]
-	pub fn pre_migrate<T: Config>() -> Result<(), &'static str> {
-		let count = Fees::<T>::iter_values().count();
```

Comment:
I'm not sure how I can tests if these elements are removed because the storage is no longer referenced from `polkadot.js`. I'm 99.9% pretty sure it has already been removed correctly but do you know how to check it?

Reply:
@lemunozm You can either:
- Generate a valid storage key and query the raw storage on polkadotjs/apps
- Run this migration code with `try-runtime` code in your local machine to ensure that the pre_migrate fails :)

Reply:
Ok thanks!